www/src/setupTests.ts

```diff
@@ -3,3 +3,8 @@
 // expect(element).toHaveTextContent(/react/i)
```

Comment:
Nit: Are these comments necessary?

Reply:
No, they just came with create-react-app, should we remove them?

Reply:
Given that we can always go to [jest-dom](https://github.com/testing-library/jest-dom), I'm in favor of removing this